src/test/librbd/test_mock_fixture.h

```diff
@@ -25,7 +25,11 @@ ACTION_P(CopyInBufferlist, str) {
 }
 
 ACTION_P2(CompleteContext, r, wq) {
+#if defined(__clang__)
+  ContextWQ *context_wq = static_cast<ContextWQ *>(wq);
```

Comment:
@dillaman @tchaikov 
And I think it is because compiletime evaluation of:
`WillOnce(CompleteContext(0, NULL)`
Delivers a nullptr, and that is where the trouble starts?